geometric_features/features_and_tags.json

```diff
@@ -1928,11 +1928,11 @@
       "August Historical Median Sea Ice Extent":[
         "QGreenland",
         "Historical_Sea_Ice_Extent"
-      ], 
+      ],
```

Comment:
Let's exclude this file from pre-commit checks because I just added a test in #220 to make sure it's formatted as expected and the two might slightly disagree.  In particular, the `json` output seems to not put the desired newline at the end of the file.

Reply:
I went ahead and took care of this in order to get this in.  I hope that's okay.